api/src/main/resources/liquibase.xml

```diff
@@ -1825,4 +1825,15 @@
             ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
         </sql>
     </changeSet>
+    <changeSet id="hospital-core:2022-11-29-07:30" author="vic">
```

Comment:
There's a need to add these fields to the model and hbm file, and to add/update service and dao files to allow filtering/fetching by the status and uuid fields